bin/PerlDDS/Run_Test.pm

```diff
@@ -454,6 +462,29 @@ sub new {
     ++$index;
   }
 
+  if (defined $self->{configs} && defined $self->{config}) {
+      # Export what is in the config.
+      $self->{discovery} = $self->{configs}{$self->{config}}{discovery};
+      my $config = $self->{configs}{$self->{config}}{file};
+      keys %$config;
+      while (my ($s, $t) = each %$config) {
+          keys %$t;
+
+          if ($s =~ /\//) {
+              my @parts = split /\//, $s;
+              my $key = "OPENDDS_${s}";
+              my $value = "\@$parts[1]";
+              $ENV{$key} = $value;
```

Comment:
I'm pretty sure this isn't going to work in a remote-controlled cross-compiled target.

Reply:
That's true.  PerlACE TestTargets have `SetEnv` which we can use to set environment variables in this case.